feature-pack/src/main/resources/modules/system/layers/base/javax/ejb/api/main/module.xml

```diff
@@ -25,11 +25,11 @@
 <module xmlns="urn:jboss:module:1.3" name="javax.ejb.api">
     <dependencies>
         <!-- This dep is for javax.naming -->
-        <module name="javax.api" export="true"/>
-        <module name="javax.transaction.api" export="true"/>
-        <module name="javax.xml.rpc.api" export="true"/>
```

Comment:
So if the user declares an exclusion for "javax.ejb.api" we automagically add exclusions for any module it exports?

Possible, but then if they actually want, say "javax.transaction.api", they need to configure a "don't-exclude". After we develop such a config of course. ;)